lib/eslint.js

```diff
@@ -191,8 +196,12 @@ function run(console, runOptions) {
               return
             }
 
-            var issueJson = buildIssueJson(message, path)
-            STDOUT(issueJson + "\u0000\n")
+            var readableJsonStream = buildIssueJson(message, path)
+            var output = ""
```

Comment:
Shouldn't instead be like follows ?
```ruby
            var output = "" 
            readableJsonStream.on('data', (chunk) => {
              output = output + chunk
            })
            readableJsonStream.on('end', () => process.stdout.write(`${JSON.stringify(output)}` + "\u0000\n"))
```

Reply:
We want to avoid applying stringify on large jsons, that's why it's applied on the chunks, building the string little by little. Does it make sense?

Reply:
Yeap, it does.